packages/core/src/sequelize-typescript.ts

```diff
@@ -529,4 +558,67 @@ export abstract class SequelizeTypeScript {
       }
     }
   }
+
+  /**
+   * Alias of {@link AbstractQueryInterface#createSchema}
+   *
+   * @param schema Name of the schema
+   * @param options
+   */
+  async createSchema(schema: string, options?: CreateSchemaOptions): Promise<void> {
+    return this.queryInterface.createSchema(schema, options);
+  }
+
+  /**
+   * Alias of {@link AbstractQueryInterface#showAllSchemas}
+   *
+   * @param options
+   */
+  async showAllSchemas(options?: ShowAllSchemasOptions) {
+    return this.queryInterface.showAllSchemas(options);
+  }
+
+  /**
+   * Alias of {@link AbstractQueryInterface#dropSchema}
+   *
+   * @param schema
+   * @param options
+   */
+  async dropSchema(schema: string, options?: QueryRawOptions) {
+    return this.queryInterface.dropSchema(schema, options);
+  }
+
+  /**
+   * Alias of {@link AbstractQueryInterface#dropAllSchemas}
+   *
+   * @param options
+   */
+  async dropAllSchemas(options?: QueryInterfaceDropAllTablesOptions) {
```

Comment:
Is this correct?

Reply:
Good catch. The two types (this one and `QiDropAllSchemasOptions`) are equivalent but let's use the correct one. Done!